change/@ni-nimble-components-5102e613-18d0-4310-9a34-a0a1564227df.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
> A beneficial side effect of this change is that the click area of the anchor now exactly matches the hover & focus area of the menu item.

Doesn't look like the pointer matches the area though, it's subtle but you end up in a spot where it looks like you can click but nothing happens

![pointershape](https://github.com/ni/nimble/assets/1588923/8c948e49-ffd2-4907-8f02-578cf9627ecc)

Reply:
Good catch. That's specifically a problem with submenus due to the menu-item styling apply to the slotted menu. I've resolved the problem.